underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsAclProvider.java

```diff
@@ -33,4 +33,14 @@
    */
   @Nullable
   AccessControlList getAcl(FileSystem hdfs, String path) throws IOException;
+
+  /**
+   * Sets the ACL for an hdfs path.
+   *
+   * @param hdfs the HDFS client
+   * @param path the path to set the ACL for
+   * @param acl the {@link AccessControlList} representation
+   * @throws IOException if ACL can not be set or not supported
```

Comment:
if it is not supported, it should be a noop?

Reply:
done